src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -618,6 +622,20 @@ public void debug(String project, String application, String context, String com
     ExecHelper.executeWithTerminal(new File(component), false, command, "debug", "port-forward", "--local-port", port.toString());
   }
 
+  @Override
+  public boolean isServiceCatalogAvailable(OpenShiftClient client){
+    try {
+      ServiceCatalogClient sc = client.adapt(ServiceCatalogClient.class);
```

Comment:
Don't need to perform a request to check for service availability (which is bad as you call this method in Action.isVisible that will slow down the UI significantly). The Kubernetes/OpenShiftClient has a isAdaptableMethod(Class) that does the trick

Reply:
done.